keps/sig-node/3673-kubelet-parallel-image-pull-limit/README.md

```diff
@@ -385,6 +386,7 @@ A new node_e2e test with  `serialize-image-pulls==false` will be added to make s
 
 #### Beta
 - Gather feedback from developers and surveys
+- Add e2e test to cover the parallel image pull case
```

Comment:
do we have tests
- validating behavior of the same image being downloaded in parallel? What if the image marked as "pull always"?
- Bad image being stuck on download for a long time and other images still can be downloaded?

Or those are covered with the unit tests?

It will be great if you can fill up the sections above listing tests, at least high level overview.

Reply:
Added more details in the testing sections (also updated the unit test part). Please take a look.